rbac/management/group/view.py

```diff
@@ -502,7 +505,7 @@ def remove_principals(self, group, principals, account=None, org_id=None):
         return group
 
     @action(detail=True, methods=["get", "post", "delete"])
-    def principals(self, request, uuid=None):
+    def principals(self, request: Request, uuid: UUID = None):
```

Comment:
in this case you should use `Optional` from typing `package`
`def principals(self, request: Request, uuid: Optional[UUID] = None):`

Reply:
Refactored, thanks!